### ISSUE TITLE
needs update for space age

### ISSUE THREAD

**Issue by erubianwarlord** [Bilka2/AbandonedRuins#42]
the ruins mod wont work in the current 2.0 version

**Comment by keysivi** [comment]
I agree. I really liked your mod! It would be a shame if it died... 

Please update it!

**Comment by zyeborg** [comment]
Yeah Please update mate ^^
and Your Epic ruin idea will be Nice !

**Comment by keysivi** [comment]
Hello! I managed to revive your mod
"The Ruins Mod".

What should I do? Give you the corrected archive so that you can update your original release? Or make a fork myself?

I also suggest removing the content from "The Ruins Mod" and adding it as a separate mod so that players can choose which ruins they want to see in their game.

Thank you!
